.rubocop_fix_me.yml

```diff
@@ -988,6 +988,8 @@ RSpec/VerifiedDoubles:
     - 'spec/views/shared/_select_work_type_modal.html.erb_spec.rb'
     - 'spec/views/hyrax/base/_work_description.erb_spec.rb'
     - 'spec/views/hyrax/base/_form_instructions_spec.rb'
+    - 'spec/views/hyrax/base/_form.html.erb_spec.rb'
```

Comment:
I added these because trying to create some of the doubles for these objects using the proper "verified doubles" is a non trivial thing to do. This is how hyrax did their tests and im willing to stick with it for sanity sake.